pkg/ui/workspaces/cluster-ui/src/statsTableUtil/statsTableUtil.tsx

```diff
@@ -290,6 +292,43 @@ export const statisticsTableTitles: StatisticTableTitleType = {
       </Tooltip>
     );
   },
+  rowsWritten: (statType: StatisticType) => {
+    let contentModifier = "";
+    const fingerprintModifier = "";
+    switch (statType) {
+      case "transaction":
```

Comment:
question: should I remove this case if we're not supporting rowsWritten as a column on the Transactions page? Also, should I be adding this column onto the Transactions page, either in this PR or a future one?